.github/workflows/release.yml

```diff
@@ -203,6 +203,68 @@ jobs:
     strategy:
       matrix:
         aws_region: ${{fromJson(github.event.inputs.aws_region).aws_region}}
+        # We need to define the raw parameters to the matrix here but will customize each
+        # below.
```

Comment:
I think this was removed as part of Nathan's PR, might've gotten added back accidentally during the rebase: https://github.com/aws-observability/aws-otel-lambda/pull/149/files

Probably worth going through #149 quickly to make sure there weren't any other accidental overwrites!